tests/test_lpnc.py

```diff
@@ -0,0 +1,36 @@
+import canela.lpnc as lpnc
+from canela.data import data_path
+import os
+import ase
+import ase.io
+
+# test Au25(PET)18 nanocluster
+au25 = ase.io.read(os.path.join(data_path, 'au25_pet18_opt_-1.xyz'))
+
+
+def test_coreshell():
```

Comment:
needs more descriptive name, `test_coreshell__{conditions}_{should do what}`. E.g.: you could have:

test_coreshell___au25_core_is_13_au_atoms
test_coreshell___au25_shell_is_all_atoms_not_in_core

You've commented everything so this is already very clear! If you change the comments to the test names, then when a test fails, it will also be really clear what is happening.

Reply:
Improved test names (see new commit below)